symfony/index.rst

```diff
@@ -91,6 +91,7 @@ must implement ``Translation\Common\Storage``.
           output_dir: "%kernel.root_dir%/Resources/translations"
           remote_storage: ["php_translation.adapter.loco"]
           local_storage: ["app.custom_local_storage"]
+          output_format: "yml"
```

Comment:
Make it "xlf" because it is the default value. 

Also, xlf is the internet standard format for translation files.